scylla-cql/src/frame/frame_errors.rs

```diff
@@ -66,20 +64,12 @@ pub enum FrameSerializationError {
 pub enum ParseError {
     #[error("Low-level deserialization failed: {0}")]
     LowLevelDeserializationError(#[from] LowLevelDeserializationError),
-    #[error("Could not serialize frame: {0}")]
-    BadDataToSerialize(String),
     #[error("Could not deserialize frame: {0}")]
     BadIncomingData(String),
     #[error(transparent)]
-    DeserializationError(#[from] DeserializationError),
-    #[error(transparent)]
-    DeserializationTypeCheckError(#[from] TypeCheckError),
-    #[error(transparent)]
     IoError(#[from] std::io::Error),
     #[error(transparent)]
-    SerializeValuesError(#[from] SerializeValuesError),
-    #[error(transparent)]
-    SerializationError(#[from] SerializationError),
+    TryFromIntError(#[from] std::num::TryFromIntError),
 }
```

Comment:
You add `TryFromIntError` probably because some removed variant contained it. Doesn't this remove the context that the variant provided, leaving us with plain `TryFromIntError` without context?